src/main/java/hong/Hong.java

```diff
@@ -0,0 +1,69 @@
+package hong;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import parsers.Parser;
+import tasks.Task;
+import tasks.TaskList;
+import ui.Ui;
+
+
+public class Hong {
+    private static ArrayList<Task> tasks;
+
+    /*
+     * Entry point of the Hong app.
+     *
+     * @param args The command line arguments.
+     */
+    public static void main(String[] args) {
+        boolean isLoopContinuing = true;
+        Ui.sayHello();
+        TaskList taskList = new TaskList();
+        Scanner myObj = new Scanner(System.in);
+        while (isLoopContinuing) {
+            String userInput = myObj.nextLine();
+            Parser parser = new Parser();
+            String parsedCommand = parser.parseCommand(userInput);
+            switch (parsedCommand) {
```

Comment:
I like your structuring of code! It is clear for reading and maybe you can consider adding in more empty lines to separate code chunks. Just to make it easier for reading :)